packages/react-native-jigsaw/db.json

```diff
@@ -0,0 +1,3 @@
+{
+  "components": []
```

Comment:
whats this?

Reply:
This is a mock server that creates an endpoint /components mimicking what we have on the backend in order to test the collector script.